src/utils/wallet.ts

```diff
@@ -176,6 +176,29 @@ const wallet = {
     return privateKey.xpubkey;
   },
 
+  /**
+   * Get xpubkey in account derivation path from seed
+   *
+   * @param {String} seed 24 words
+   * @param {Object} options Options with passphrase, networkName and accountDerivationIndex
+   *
+   * @return {String} Wallet xpubkey
+   * @memberof Wallet
+   * @inner
+   */
+  getXPubKeyFromSeed(seed: string, options: { passphrase?: string, networkName?: string, accountDerivationIndex?: string } = {}): string {
+    const methodOptions = Object.assign({passphrase: '', networkName: 'mainnet', accountDerivationIndex: '0\''}, options);
+    const { passphrase, networkName, accountDerivationIndex } = methodOptions;
+
+    const network = new Network(networkName);
+    const code = new Mnemonic(seed);
+    const xpriv = code.toHDPrivateKey(passphrase, network.bitcoreNetwork);
+    // We have a fixed derivation until the coin index
+    // after that we can receive a different account index, which the default is 0'
+    const privkey = xpriv.derive(`m/44'/${HATHOR_BIP44_CODE}'/${accountDerivationIndex}`);
```

Comment:
I think we are doing two things here. Maybe we should split this method into `getXPrivKeyFromSeed` and call this new method from here. What do you think?

Reply:
I agree. I created a method to get the xpriv of the root, and then in this method I derive. What do you think?

Reply:
Great. Thanks!